products/productController.js

```diff
@@ -0,0 +1,50 @@
+const productModel = require("../model/products");
+
+const createProduct = async (req, res, next) => {
```

Comment:
the router should be written  inside the route file,

and then, you are not to work on the create user endpoint